Examples/GetFields/show_fields.py

```diff
@@ -0,0 +1,31 @@
+############################################################
+# Step 1)  Use Datasource object from the Document API
+############################################################
+from tableaudocumentapi import Datasource
+
+############################################################
+# Step 2)  Open the .tds we want to inspect
+############################################################
+sourceTDS = Datasource.from_file('World.tds')
+
+############################################################
+# Step 3)  Print out all of the fields and what type they are
+############################################################
+print('----------------------------------------------------------')
+print('--- {} total fields in this datasource'.format(len(sourceTDS.fields)))
+print('----------------------------------------------------------')
+count = 0
+for field_key, field in sourceTDS.fields.items():
```

Comment:
If you never use field_key I think this could be simplified to:
`for count, field in enumerate(sourceTDS.fields.values():`